RevenueCatUI/Data/TestData.swift

```diff
@@ -256,6 +255,11 @@ internal enum TestData {
     )
     static let paywallHeaderImageName = "9a17e0a7_1689854430..jpeg"
     static let paywallBackgroundImageName = "9a17e0a7_1689854342..jpg"
+    static let images: PaywallData.Configuration.Images = .init(
+        header: Self.paywallHeaderImageName,
+        background: Self.paywallBackgroundImageName,
+        icon: Self.paywallHeaderImageName
+    )
```

Comment:
This is much better than the array solution. Thanks @guido732 @lburdock for the idea.